next-cloudinary/src/components/CldVideoPlayer/CldVideoPlayer.tsx

```diff
@@ -131,8 +131,17 @@ const CldVideoPlayer = (props: CldVideoPlayerProps) => {
         }
       }
 
+      /* 
+      Parse the value passed to 'autoplay';
+      if its a boolean set it directly to browser standard prop autoplay else fallback to default; 
+      if its a string set it to cloudinary video player autoplayMode prop else fallback to default;
+      */
+      const autoPlayValue = typeof autoPlay === 'boolean'  ? autoPlay : false;
```

Comment:
i wondrer if we need to add a test here for `autoplay="true"` which i believe would pass it in as a string, to avoid people who don't quite understand the difference having an issue

Reply:
Okay just to be clear are you talking about people mistakenly passing `autoplay="true"` as a string instead of `autoplay=true` as a boolean ?.

if that's what you meant what do you think about giving `autoPlay` a new type of  `AutoPlayType | boolean`, 
where `AutoPlayType` would be:
````
type AutoPlayType = "never" | "always" | "on-scroll"
````

I believe this implementation will help make sure people using the component know what type of string value to pass.

Tho this is just a thought i would like to hear what you think about it and if you have any other better approach.

Reply:
my only thinking has been that i've somewhat avoid adding values like that, only because they could potentially change in the API and would always require that i update it, but i think having better Types would probably be better than taking a little time to update to support a new one, do you agree?

i still like the idea though of allowing someone to pass in prop="true". though it's not technically correct, it still somewhat feels ergonomically okay, considering native HTML allows you to do so, for instance: https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML

Reply:
You really do have a valid point about not adding values like that as they could change in the API.

i will implement the `prop="true"` feature right up